tsconfig.json

```diff
@@ -15,16 +15,16 @@
 		],
 		"module": "umd",
 		"moduleResolution": "node",
-		"noImplicitAny": true,
-		"noImplicitThis": true,
 		"noUnusedLocals": true,
 		"noUnusedParameters": true,
 		"outDir": "./_build",
 		"preserveConstEnums": true,
 		"removeComments": true,
 		"rootDir": ".",
 		"sourceMap": true,
-		"target": "ES5"
+    "strict": true,
+		"target": "es5",
```

Comment:
Because we use async/await I'm not sure how we're able to build this targeting es5 without downlevelIteration.

Reply:
TS down emits `async/await` for `"es5"`. `downlevelIteration` is only for iterators.

Reply:
haha I was thinking async generators! that must've been confusing, sorry!